build/_k-test.shade

```diff
@@ -58,7 +59,10 @@ projectFile=''
                     testArgs = " -parallel none";
                 }
 
-                Dnx("test" + testArgs, projectFolder);
+                if (!IsLinux || string.IsNullOrEmpty(KOREBUILD_TEST_SKIPMONO))
```

Comment:
Check if `1` of `true`, not that that if it's empty or not. Otherwise, setting it to "false" or "foo" will skip the tests too